quickinfo/quickinfo-library/src/main/java/org/silverpeas/components/quickinfo/model/DefaultQuickInfoService.java

```diff
@@ -213,6 +217,60 @@ public News create(final News news) {
     return savedNews;
   }
 
+  @Override
+  @Transactional
+  public News copyNews(final News newsToCopy, final PasteDetail pasteDetail) {
+
+    // Initializing the news instance
+    final News news = News.builder(newsToCopy).build();
+    news.setDraft();
+    news.setComponentInstanceId(pasteDetail.getToComponentId());
+    news.setCreatorId(pasteDetail.getUserId());
+
+    // Creating linked publication
+    PublicationDetail publication = news.getPublication();
+    publication.setIndexOperation(IndexManager.NONE);
+    final PublicationPK pubPK = getPublicationService().createPublication(publication);
+    publication = getPublicationService().getDetail(pubPK);
+
+    // Saving the new news instance into repository
+    news.setPublicationId(pubPK.getId());
+    news.setPublication(publication);
+    final News savedNews = newsRepository.save(news);
+
+    // Copying decorators
+    final ResourceReference pubSourceRef = newsToCopy.getPublication().getPK().toReference();
+    final ResourceReference pubDestRef = news.getPublication().getPK().toReference();
+    // - PDC
+    copyPdcPositions(newsToCopy, savedNews);
```

Comment:
Yes, I had a chat with David about this. During the development, I realised that there was no particular reason not to copy the PDC positions. What's more, it would introduce a difference in behaviour to copying a publication into the kmelia component and get potential misunderstandings on the user side.

Reply:
Ok